.github/workflows/build_and_test.yml

```diff
@@ -30,10 +29,28 @@ jobs:
         run: npm ci
 
       - name: Build Web App
-        run: npm run action gulp -- build browser
+        run: npm run action gulp build browser
```

Comment:
Shall we also update the [command line in browser section in README](https://github.com/Jigsaw-Code/outline-client#building-the-web-app) as well?

If there are too many changes, I can do it in a separate PR.

Reply:
Yeah i can update that, this just was never needed to begin with